src/management/configuration/portal/portal.html

```diff
@@ -107,6 +107,15 @@ <h3>OpenAPI viewers</h3>
         </tr>
       </table>
 
+      <h3>Labels</h3>
+      <md-input-container class="md-block">
+        <label>Labels</label>
```

Comment:
Is it possible to add some description to explain what is the purpose of this field ? (like api-key header)

Reply:
resolved